src/attribute.cpp

```diff
@@ -1,5 +1,5 @@
-#include "packet.h"
-#include "attribute.h"
+#include "../include/radproto/packet.h"
```

Comment:
Since you added `target_include_directories`, relative paths are unnecessary here and below.